.fixtures.yml

```diff
@@ -4,12 +4,8 @@ fixtures:
     compliance_markup: https://github.com/simp/pupmod-simp-compliance_markup
     augeasproviders_core: https://github.com/simp/augeasproviders_core
     augeasproviders_grub: https://github.com/simp/augeasproviders_grub
-    concat: https://github.com/simp/puppetlabs-concat
-    logrotate: https://github.com/simp/pupmod-simp-logrotate
-    pki: https://github.com/simp/pupmod-simp-pki
```

Comment:
Need to readd this to the fixtures since it's required by rsyslog when you run that part of the acceptance test.

Reply:
I think concat and logrotate were here for rsyslog too now that I think about it